emperor/support_files/js/emperor.js

```diff
@@ -1815,7 +1818,19 @@ function changeAxesDisplayed() {
   // 		}
 	// }
 
-	var checkedboxes = [];
+    comparisonPositionlength = Object.keys(g_comparisonPositions).length
```

Comment:
This is not gonna work cause g_spherePositions is not an object of objects any more. I think we need to change the loop to loop on g_spherePositions and then check if g_comparisonPositions has the same [sampleKey][j].